pages/common/sbuild.md

```diff
@@ -0,0 +1,32 @@
+# sbuild
+
+> Build a Debian binary package in a clean `chroot` environment.
+> More information: <https://wiki.debian.org/sbuild>.
+
+- Build the package in the current directory:
+
+`sbuild`
+
+- Build the given package:
+
+`sbuild {{package}}`
+
+- Build for a certain distribution:
+
+`sbuild --dist {{distribution}}`
+
+- Build using custom dependencies:
+
+`sbuild --extra-package {{path/to/deb_files}}`
+
+- Run a shell in case of build failure to further investigate:
+
+`sbuild --build-failed-commands=%SBUILD_SHELL`
```

Comment:
```suggestion
`sbuild --build-failed-commands %SBUILD_SHELL`
```

Reply:
The docs seem to suggest using `=` for this command so IG we can leave this as it is and merge this PR.